src/codegen/ast_interpreter.cpp

```diff
@@ -265,6 +270,11 @@ void ASTInterpreter::initArguments(int nargs, BoxedClosure* _closure, BoxedGener
     }
 }
 
+// Map from stack frame pointers for frames corresponding to ASTInterpreter::execute() to the ASTInterpreter handling
+// them. Used to look up information about that frame. This is used for getting tracebacks. I think it's also used for
+// CPython introspection (sys._getframe & co).
+//
+// TODO: get kmod to check this is an accurate description
```

Comment:
sgtm.  looks like it's currently used for those things as well as part of gc scanning.